doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1415,9 +1415,13 @@ A `subject` contains the following subfields:
 
 A `notification` object contains the following subfields:
 
-+ `attrs`: List of attributes to be included in the notification message. If not specified (or empty), all attributes are included
-  in the notification. It also defines the order in which attributes will appear in notifications when `attrsFormat` `value` is used
-  (see "Notification messages" section).
++ `attrs` or `exceptAttrs` (both cannot be used at the same time):
+  + `attrs`: List of attributes to be included in the notification message. It also defines the order in which attributes will
```

Comment:
As always, I propose to not use future:

    It also defines the order in which attributes appear in notifications ...